# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/public/libcss/gcc33-hotfix.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -urN -x '*~' libcss-0.1.0/src/css_cat.c libcss-0.1.0-gcc-3.3.1/src/css_cat.c
--- libcss-0.1.0/src/css_cat.c	2001-02-13 22:59:27.000000000 +0100
+++ libcss-0.1.0-gcc-3.3.1/src/css_cat.c	2003-08-19 22:43:44.000000000 +0200
@@ -60,14 +60,14 @@
 
 #define STCODE(p,a,b,c,d) ((p)[0] == a && (p)[1] == b && (p)[2] == c && (p)[3] == d)
 
-u_long long offset = 0x0;
-u_long long offset_end = 0x249a4;
+unsigned long long offset = 0x0;
+unsigned long long offset_end = 0x249a4;
 
 /**
  *
  */
 
-static void super_seek (int fd, u_long long offset)
+static void super_seek (int fd, unsigned long long offset)
 {
 	int i;
 
